docs/usage/java.md

```diff
@@ -53,3 +100,16 @@ module.exports = {
   ],
 };
 ```
+
+One can also overwrite the repositories to use for version lookup through configuration.
```

Comment:
```suggestion
You can overwrite the repositories to use for version lookup through configuration.
```

Simplify, and use `you` instead of the more generic `one can`